server/common/rest.py

```diff
@@ -179,14 +180,21 @@ def diffexp_obs_post(request, data_adaptor):
 
 
 def layout_obs_get(request, data_adaptor):
+    fields = request.args.getlist("layout-name", None)
+    num_columns_requested = len(data_adaptor.get_embedding_names()) if len(fields) == 0 else len(fields)
+    if data_adaptor.config.exceeds_limit("column_request_max", num_columns_requested):
```

Comment:
Wouldn't the limit be the min of `column_request_max` and the number of embeddings given by the `data_adaptor`?

Reply:
not for this check - this is to simply do a resource check.  The existence check for the embedding comes later.

I'd like to keep these concerns separate for clarity.